### PR TITLE
Decode byte

### LINKED ISSUE #3798: base64.decodestring was deprecated and removed in Python 3.9 in favor of decodebytes
### Steps to Reproduce

It should be decodestring in python 2 and decodebytes in python 3.

```
openlibrary/plugins/importapi/code.py
462:        username, password = base64.decodestring(authstring).split(':')
606:        username, password = base64.decodestring(authstring).split(':')
```

### PR DESCRIPTION
**Pull request by Abhirup-99**
Closes #3798

fix
### Technical

Removes decodestring and adds decodebytes which is it's replacement.

### CODE REVIEW

File: openlibrary/plugins/importapi/code.py
Comment (tirkarthi) [inline openlibrary/plugins/importapi/code.py:462]: The project seems to support Python 2 though this code path might not be tested. `decodestring` has to be used in Python 2 and `decodebytes` has to be used in Python 3 to maintain compatibility.

Comment (Abhirup-99) [reply openlibrary/plugins/importapi/code.py:462]: Can you suggest what I should I do over here?

**Review by tirkarthi — APPROVED** [review]
LGTM. Thanks.

**Comment by Abhirup-99** [comment]
@tirkarthi when can this be merged? and should I squash everything into 1 commit?

**Comment by tirkarthi** [comment]
authstring seems to be str from the context. You might want to encode it to bytes if string while passing to decodebytes.

**Comment by tirkarthi** [comment]
I am not a member of the repo. You might want to wait on maintainers for approval.

**Comment by Abhirup-99** [comment]
Ok thanks a lot, for the suggestions and the info.

**Comment by cclauss** [comment]
What is the status of this?

**Comment by Abhirup-99** [comment]
I am pushing another. It got messed up in some conflicts.

**Pull request closed without merging**
